docs/add-new-token.md

```diff
@@ -0,0 +1,158 @@
+# Adding New Token to the List
+
+The goal of this guide is walk you through the steps needed to add a new token to the token list.
+
+## Before you begin
+
+You need to have some informations about the token you want to add to the list before you begin this tutorial and they are:
+
+1. Chain ID
+
+The new token can be added to testnet and/or mainnet and the chain ID for each is the following:
+
+```json
```

Comment:
Should have used "text" instead of "json"? Could have used a list instead of a code block too.

Reply:
fixed